pkg/controller/secret/secret_controller.go

```diff
@@ -379,8 +404,14 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 		watchdogURL = readSecretKey(r, &request, secretNameDMS, secretKeyDMS)
 	}
 
+	// grab the console URL for PD alerts
+	consoleUrl, err := r.getWebConsoleUrl()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
```

Comment:
If we're not able to get the console URL it shouldn't block configuration of alertmanager.  We can still get stuff done, we just are not quite as efficient.  But not having alerts configured is a big problem.  Recommend we simply ignore the error and set consoleUrl to something that won't cause the config to blow up.